snap/snapcraft.yaml

```
@@ -173,18 +173,19 @@ parts:
       source scripts/activate.sh
       set -u
 
-      # Copy and replace the application files
-      cp -vr $CRAFT_PART_BUILD/lighting-common/* examples/lighting-app/lighting-common/
-      cp -vr $CRAFT_PART_BUILD/linux/* examples/lighting-app/linux
+      # Add the App to be build from CHIP SDK's build system
+      # See https://github.com/project-chip/connectedhomeip/issues/32925
+      cp -vr $CRAFT_PART_BUILD/app examples/
```

I suggest giving the `app` directory a more distinct name, such as `gpio-app` to make it clearer that app/linux/BUILD.gn (line 46) is using the common code taken from this repo. This should also be consistent with the binary name.

```suggestion
      cp -vr $CRAFT_PART_BUILD/app examples/gpio-app
```